spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/util/InetIPv6Util.java

```diff
@@ -93,10 +72,11 @@ else if (address != null) {
 							InetAddress inetAddress = addrs.nextElement();
 							if (inetAddress instanceof Inet6Address
 									&& !inetAddress.isLoopbackAddress()
+									&& !isLocalAddress(inetAddress)
```

Comment:
You can directly use `inetAddress.isLinkLocalAddress()`.

Reply:
done